config/config.json

```diff
@@ -145,7 +145,9 @@
         "UsernameAttribute": "",
         "IdAttribute": "",
         "SkipCertificateVerification": false,
-        "QueryTimeout": 60
+        "QueryTimeout": 60,
+        "LoginFieldName": "",
```

Comment:
The whole system console is about to be refactored anyways so I'd rather not add a Branding section right now (I'm pretty sure the Branding spec purposefully placed it in Team Settings temporarily). It's also the customizing the look of LDAP part of the software :P I don't really see it as branding